museum_of_dreams_project/urls.py

```diff
@@ -41,8 +43,44 @@
         distill_path(
             "", views.HomeView.as_view(), name="home", distill_file="index.html"
         ),
-        distill_path("films/list", views.FilmListView.as_view(), name="film_list"),
-        distill_path("films/<pk>", views.FilmDetailView.as_view(), name="film_detail"),
+        path("favicon.ico", RedirectView.as_view(url="static/admin/img/favicon.ico")),
+        path("mentions-api", views.MentionsApiView.as_view(), name="mentions_api"),
+        # website pages
+        distill_path(
+            "films/list",
```

Comment:
I think just films/ rather than films/list would be better, because there's nothing at films/ at the moment.